discovery-infra/tests/base_test.py

```diff
@@ -68,21 +70,10 @@ def cluster(self, api_client, request, nodes):
         clusters = []
 
         @JunitTestCase()
-        def get_cluster_func(cluster_name: Optional[str] = None,
-                             additional_ntp_source: Optional[str] = consts.DEFAULT_ADDITIONAL_NTP_SOURCE,
-                             openshift_version: Optional[str] = env_variables['openshift_version'],
-                             user_managed_networking: Optional[bool] = env_variables["user_managed_networking"],
-                             high_availability_mode=env_variables["high_availability_mode"],
-                             olm_operators=env_variables['olm_operators']):
-            if not cluster_name:
-                cluster_name = env_variables.get('cluster_name', infra_utils.get_random_name(length=10))
-            res = Cluster(api_client=api_client,
-                          cluster_name=cluster_name,
-                          additional_ntp_source=additional_ntp_source,
-                          openshift_version=openshift_version,
-                          user_managed_networking=user_managed_networking,
-                          high_availability_mode=high_availability_mode,
-                          olm_operators=olm_operators)
+        def get_cluster_func(cluster_config: DefaultClusterConfig):
```

Comment:
DefaultClusterConfig -> ClusterConfig

Reply:
ClusterConfig is the base cluster config (it will be abstract soon). 
DefaultClusterConfig is another abject with default values inside

Reply:
but you should expect base class and not default one